Modules.md

```diff
@@ -108,8 +108,9 @@ to allow *explicitly* sharing linear memory between multiple modules.
 ## Initial state of linear memory
 
 A module will contain a section declaring the linear memory size (initial and
-maximum size allowed by `sbrk`) and the initial contents of memory (analogous
-to `.data`, `.rodata`, `.bss` sections in native executables).
+maximum size allowed by [`resize_memory`](AstSemantics.md#resizing) and the
+initial contents of memory (analogous to `.data`, `.rodata`, `.bss` sections in
+native executables).
```

Comment:
In a sense, `.rodata` *is* letting wasm handle them but restricting where they can go.  Having the data declared in a global section also allows the wasm compiler to bake in more constant address info compared to a fully dynamic API that returned dynamic pointers to the data (both with and without dynamic loading).  That all being said, there's a lot more design space to consider on this subject, but it seems like a topic for a separate PR/issue.